resources/letters/attributes/ltr_size.yml

```diff
@@ -0,0 +1,10 @@
+type: string
+
+enum:
+  - us_legal
+
+description: >
+  Specifies the size of the letter. Currently only applicable for legal letters.
+  To use `us_legal`, set the [Lob-Version header](#tag/Versioning-and-Changelog) to `2024-01-01`. Do not include this attribute as well the header version set to 2024-01-01 if you are not sending a legal letter.
```

Comment:
can this be a little bit clearer?

Reply:
```suggestion
  To use `us_legal`, set the [Lob-Version header](#tag/Versioning-and-Changelog) to `2024-01-01`. Do not include neither this attribute nor the header version set to 2024-01-01 if you are not sending a legal letter.
```

Reply:
@juanfriss updated the verbiage to be less confusing.